### ISSUE TITLE
The vscode IDF plugin cannot use the MSYS2 default terminal by clicking the monitor button(IDFGH-13393)

### ISSUE THREAD

**Issue by JDingOneSelf** [espressif/esp-idf#14301]
### Answers checklist.

- [X] I have read the documentation [ESP-IDF Programming Guide](https://docs.espressif.com/projects/esp-idf/en/latest/) and the issue is not addressed there.
- [X] I have updated my IDF branch (master or release) to the latest version and checked that the issue is present there.
- [X] I have searched the issue tracker for a similar issue and not found a similar issue.

### General issue report

xxxx@LAPTOP-8EUDJHTE  /d/xxxxx/01Work/01Demo/01ESP32S3_Demo/blink
$ & set IDF_PATH='d:\Program\Espressif\IDF\v4.4.5\esp-idf'
bash: syntax error near unexpected token `&'

xxxx@LAPTOP-8EUDJHTE  /d/xxxxx/01Work/01Demo/01ESP32S3_Demo/blink
$ & 'd:\Program\Espressif\445\python_env\idf4.4_py3.8_env\Scripts\python.exe' 'd:\Program\Espressif\IDF\v4.4.5\esp-idf\tools\idf_monitor.py' -p COM4 -b 115200 --toolchain-prefix xtensa-esp32s3-elf- --target esp32s3 'd:\xxxxx\01Work\01Demo\01ESP32S3_Demo\blink\build\blink.elf' 
bash: syntax error near unexpected token `&'

**Comment by brianignacio5** [comment]
Are you talking about https://github.com/espressif/vscode-esp-idf-extension ?

Why are you using MSYS2 ? Why don't use Powershell or Command Prompt (cmd.exe) ?

**Comment by JDingOneSelf** [comment]
Yes, I am used to using MSYS2, and PowerShell and cmd have garbled characters, is there a solution?It worked before

**Comment by brianignacio5** [comment]
This has been fixed in the latest release of https://github.com/espressif/vscode-esp-idf-extension 

Please give it a try.

**Comment by Alvin1Zhang** [comment]
Thanks for reporting, will close now, feel free to create a new ticket in https://github.com/espressif/vscode-esp-idf-extension, thanks.
